apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx

```diff
@@ -863,68 +880,89 @@ export const AttributeSelectionOIDC: FunctionComponent<AttributeSelectionOIDCPro
-                                        <Trans
-                                            i18nKey={
-                                                "console:develop.features.applications.edit.sections.attributes." +
-                                                "selection.attributeComponentHintAlt"
-                                            }
-                                        >
-                                        Manage the user attributes you want to share with this application.
-                                        You can add new attributes and mappings by navigating to
-                                            <Link
-                                                external={ false }
-                                                onClick={ () => {
-                                                    history.push(
-                                                        AppConstants.getPaths()
-                                                            .get("LOCAL_CLAIMS")
-                                                    );
-                                                } }
-                                            > Attribute.
-                                            </Link>
-                                        </Trans>
-                                    </Hint>
-                                )
-                            }
+                                    <Link
+                                        external={ false }
+                                        onClick={ () => {
+                                            history.push(
+                                                AppConstants.getPaths()
+                                                    .get("ATTRIBUTE_MAPPINGS")
+                                                    .replace(":type", ClaimManagementConstants.OIDC)
+                                            );
+                                        } }
+                                    > Attributes.
+                                    </Link>
+                                </Trans>
+                            </Hint>
+                        </Grid.Column>
+                    </Grid.Row>
+                    <Grid.Row className="mt-5">
+                        <Grid.Column computer={ 16 } tablet={ 16 } largeScreen={ 12 } widescreen={ 12 }>
+                            <Grid.Row>
+                                <Form>
+                                    <Grid.Column>
+                                        <Form.Field>
+                                            <div className="display-flex">
+                                                <CopyInputField
+                                                    className="copy-input spaced"
+                                                    value={ selectedScopes.join(" ") }
+                                                    data-testid={ `${ testId }-client-id-readonly-input` }
```

Comment:
update the test-id to component id

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3640/commits/1eea89c4b4ff909d642ad66bb648e32a18e60490